### ISSUE TITLE
[docs] SmoothL1Loss to document behavior at beta=0

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#68558]
https://pytorch.org/docs/stable/generated/torch.nn.SmoothL1Loss.html#torch.nn.SmoothL1Loss:

The docs only specify what happens when beta tends to zero (beta -> 0), it also specifies that beta must be non-negative.

What happens when beta = 0? Does it compute L1 loss in this case?

I think it would be useful to cover this case explicitly in the docs

cc @brianjo @mruberry @albanD @jbschlosser @walterddr

**Comment by jbschlosser** [comment]
Yes, when beta = 0, SmoothL1Loss computes L1Loss exactly. We'd accept a PR making the wording explicit for this case if you want to take a crack at it.
